### ISSUE TITLE
Questions about network training

### ISSUE THREAD

**Issue by modestlyh** [NVlabs/neural_rx#9]
Evaluating the model is done using a fixed SNR every N times, this way of using a single point maybe lead to upward curvature at high SNRs, normally models should be evaluated using data that the model hasn't seen before, so I find it strange why a single SNR is used instead of randomly sampling in another small range

**Comment by SebastianCa** [comment]
Hi @modestlyh,

I assume you’re referring to the evaluation during the training of the neural receiver? During training, we fix the SNR to ensure that the loss does not fluctuate due to varying SNR values.

For the final evaluation (see the example notebooks), we assess the performance over the entire SNR range.

**Comment by modestlyh** [comment]
@SebastianCa Yes。Evaluate every N iterations, if N is small, I set it to 300, I can observe that the loss of a single SNR point floats more significantly, so I think this approach requires N to be large, N in Config is 1000, which also means that the total number of iterations has to be large。

**Comment by SebastianCa** [comment]
Keep in mind that the validation loss is only for your information and has no impact on the training performance itself (there is no early stopping, etc.). Since we evaluate the BLER curves after training, this approach sounds fine to me.

Of course, you can implement any other sampling strategy for the validation SNR if needed.

I’ll close this issue, as it is not a problem with the code.
